### DISCUSSION TITLE
Deployment fails when tensorflow-cpu is present in requirements

### DISCUSSION THREAD

**Discussion by hosenur** [deta/cloud-docs#302] [Help] [answered]
I have a  simple fastAPI back-end which has tensorflow-cpu as a requirement its size is about ~250MB, the deployment fails with message 
```
Deploying...
Successfully deployed changes
Updating dependencies...
Error: failed to update dependencies: Internal server error
```
Is it a size limit which is preventing the deployment or it's something else

**Comment by jnsougata** [comment, marked as answer]
yes the resource size is the issue here
